.storybook/preview.jsx

```diff
@@ -1,21 +1,23 @@
-const React = require("react")
-const { ChakraProvider } = require("@chakra-ui/react")
-import { getTheme } from "../src/ui/theme"
-
-import '@nypl/design-system-react-components/dist/styles.css'
+const React = require('react');
+const { ChakraProvider } = require('@chakra-ui/react');
+import { getTheme } from '../src/ui/theme';
+import '@nypl/design-system-react-components/dist/styles.css';
 
 // https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters
 export const parameters = {
   // https://storybook.js.org/docs/react/essentials/actions#automatically-matching-args
   actions: { argTypesRegex: '^on.*' },
+  options: { showPanel: true },
 };
 
 const withChakra = (StoryFn, _context) => {
+  const { args } = _context;
```

Comment:
Just fyi I put the _ in names usually when a variable is not being used, so in this case you can remove the _ because it's not being used. But not a big deal.